__tests__/app.test.js

```diff
@@ -22,3 +22,31 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with all of the topics in the database", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics).toHaveLength(3);
+        topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
+      });
+  });
+
+  test("404: Responds with a not found error message", () => {
+    return request(app)
+      .get("/api/not-a-route")
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("not found");
+      });
+  });
+});
```

Comment:
Nice job with this error test - but I would put it in it's own describe block. It doesn't belong within the /api/topics one

Reply:
Would you suggest having all error handling tests for all endpoints grouped together under a single 'Error Handling' describe block, rather than paired with the 'happy path' testing for that endpoint?